GS_example.py

```diff
@@ -17,9 +17,13 @@
 if __name__ == "__main__":
     params = PropagationParams.get_example_propagation_data()
 
+    # Choose proper propagation parameters
     params.sigma = 4
     params.matrix_size = 256
     params.pixel = 0.9
+
+    # Define target optical field and input amplitude
+    # In this example two focal points placed outside of the main optical axis
     x_shift1 = 50
     x_shift2 = 25
```

Comment:
units?

Reply:
For now we can use comments :)

Reply:
I added simple description of all parameters in params.py